modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/messaging/SynchronizeOccurrenceStatusListener.java

```diff
@@ -0,0 +1,43 @@
+package com.liferay.powwow.meetings.messaging;
+
+import com.liferay.portal.kernel.messaging.BaseMessageListener;
+import com.liferay.portal.kernel.messaging.Message;
+import com.liferay.powwow.model.PowwowMeetingOccurrence;
+import com.liferay.powwow.occurrence.OccurrenceStatus;
+import com.liferay.powwow.service.PowwowMeetingOccurrenceLocalServiceUtil;
+import java.util.List;
+
+public class SynchronizeOccurrenceStatusListener extends BaseMessageListener{
+
+	@Override
+	protected void doReceive(Message message) throws Exception {
+		updateOccurrenceStatus();
+	}
+
+	private void updateOccurrenceStatus() {
+		int count =
+			PowwowMeetingOccurrenceLocalServiceUtil
+				.countByStatusAndEndTimeGT(
```

Comment:
since we use comparator `>=`, then the method name should be `countByStatusAndEndTimeGE` instead?